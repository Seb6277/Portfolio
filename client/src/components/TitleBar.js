import React from 'react'
import {Navbar} from 'reactstrap'
import '../css/TitleBar.scss'

class TitleBar extends React.Component{
  render() {
    return(
      <Navbar className="title_bar" color="dark" dark expand="md">
        <h1 color="dark">{this.props.title.toUpperCase()}</h1>
      </Navbar>
    )
  }
}

export default TitleBar;