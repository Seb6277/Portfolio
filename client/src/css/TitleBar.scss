.title_bar{
    display: block !important;
    text-align: center;
    h1{
        color: white;
        font-size: 2em;
    }
}