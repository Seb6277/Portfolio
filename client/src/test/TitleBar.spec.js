import TitleBar from '../components/TitleBar'

describe('<TitleBar/>', () => {
  it('should render without crash', () => {
    const wrapper = shallow(<TitleBar/>)
    expect(wrapper).toMatchSnapshot()
  });
});