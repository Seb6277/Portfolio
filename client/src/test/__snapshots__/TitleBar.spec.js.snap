// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TitleBar/> should render without crash 1`] = `
<Navbar
  className="title_bar"
  color="dark"
  dark={true}
  expand="md"
  tag="nav"
>
  <NavbarBrand
    className="mr-auto"
    href="/"
    tag="a"
  >
    Portfolio
  </NavbarBrand>
  <NavbarToggler
    onClick={[Function]}
    tag="button"
    type="button"
  />
  <Collapse
    appear={false}
    enter={true}
    exit={true}
    in={false}
    isOpen={false}
    mountOnEnter={false}
    navbar={true}
    onEnter={[Function]}
    onEntered={[Function]}
    onEntering={[Function]}
    onExit={[Function]}
    onExited={[Function]}
    onExiting={[Function]}
    tag="div"
    timeout={350}
    unmountOnExit={false}
  >
    <Nav
      className="ml-auto"
      navbar={true}
      tag="ul"
      vertical={false}
    >
      <NavItem
        tag="li"
      >
        <NavLink
          href="/"
          tag="a"
        >
          Accueil
        </NavLink>
      </NavItem>
      <NavItem
        tag="li"
      >
        <NavLink
          href="/projets"
          tag="a"
        >
          Projets
        </NavLink>
      </NavItem>
      <NavItem
        tag="li"
      >
        <NavLink
          href="/cv"
          tag="a"
        >
          CV
        </NavLink>
      </NavItem>
      <NavItem
        tag="li"
      >
        <NavLink
          href="/contact"
          tag="a"
        >
          Contact
        </NavLink>
      </NavItem>
    </Nav>
  </Collapse>
</Navbar>
`;
